utils/update_status_batch.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import os
+import sys
+
+import requests
+from get_signals import GetAccessToken
+
+"""
+update_status_batch.py - Batch update signal status script
+
+Expects a CSV file with status updates as input, with three values per row; the signal id, the new
+status (m, o, e, i, or ...) and the status change text.
+"""
+
+EMAIL = ''
+PASSWORD = ''
+ACCEPTANCE = True
+API_URL = 'http://localhost:8000/signals'
+
```

Comment:
Are we running the script locally? Or on the server where we want to do the update?

Maybe this can be changed to be an environment variable with a default to localhost? Or perhaps an argument that can be passed to the script.

Reply:
You're right. Just kept the script simple, as it's just a one-off.

I'll add the env vars. :)